.gitignore

```diff
@@ -17,4 +17,7 @@ yalc.lock
 npm-debug.log
 package-lock.json
 yarn-error.log
-.DS_Store
\ No newline at end of file
+.DS_Store
+
+example/deps/
+sampledata/
```

Comment:
What is generating data at these two locations? Is it something you're using for local testing?

I'm okay with having exclude rules for your local setup, but can we throw a comment above them if they're specific to you or suggested locations for sample data? `# Common Example Data Directories`

Reply:
example/deps contains the index.umd.js and index.umd.js.map generated when building the Viewer `yarn install`

sampleldata is generated by `yarn install` under `example` folder and it is used from example/index.html

Good catch. Including a comment at .gitignore